### PR TITLE
Multiplier updates to plug load measure

### PR DESCRIPTION
**Pull request by joseph-robertson**
## Pull Request Description
This pull request separates out multipliers meant to adjust total energy and multipliers meant to adjust diversity of the results.  The ResidentialMiscPlugLoads argument `mult` has been split into two multipliers (`energy_mult` and `diversity_mult`). 

The energy multiplier now allows the plug load energy use to vary with building type and RECS 2015 definition of Census Division (the Mountain division is split into North and South).

The `diversity_mult` argument is simply the same as the previous definition of `mult`.

**Companion PR:** resstock-estimation [#86](https://github.com/NREL/resstock-estimation/pull/86)

### Housing Characteristics Updates

Renamed:
- Plug Loads.tsv renamed to Plug Load Diversity.tsv. The purpose of the housing characteristic is to apply a diversity multiplier.

New
- Plug Loads.tsv is now a housing characteristic that assigned energy multipliers based on building type and the RECS 2015 Census Division definition.

### ResidentialMiscPlugLoads Updates
Renames:
- `mult` to `energy_mult`

Adds:
- `diversity_mult`

Other updates:
- Argument description for `energy_mult`

## Checklist

Not all may apply:

- [ ] Unit tests have been added or updated
- [ ] All rake tasks have been run, and pass
- [ ] Documentation has been modified appropriately
- [ ] Any new options are added to `project_testing`
- [ ] `project_testing` runs without any failures
- [ ] No unexpected regression test changes
- [ ] All tests are passing (green) on circleci
- [ ] The [changelog](https://github.com/NREL/OpenStudio-BuildStock/blob/master/CHANGELOG.md) has been updated appropriately
- [ ] This branch is up-to-date with master

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Features
 - Update foundation type from the [Building Foundation Design Handbook](https://www.osti.gov/biblio/6980439-building-foundation-design-handbook) published in 1988 to RECS 2009 ([#492](https://github.com/NREL/OpenStudio-BuildStock/pull/492))
 - Synchronize weather between ResStock and ComStock which increases the number of weather stations from 215 to 941 ([#507](https://github.com/NREL/OpenStudio-BuildStock/pull/507))
 - Update Occupants per unit from RECS 2015 to PUMS 5-yr 2017 ([#509](https://github.com/NREL/OpenStudio-BuildStock/pull/509]))
-- Include additional "diversity" multiplier in plug load equations ([#TODO](TODO))
+- Allow for plug load energy consumption to vary by Census Division and nclude additional "diversity" multiplier in plug load equations ([#511](https://github.com/NREL/OpenStudio-BuildStock/pull/51))
```
Comment (joseph-robertson) [inline CHANGELOG.md:14]: "nclude" to "include"

Comment (joseph-robertson) [inline CHANGELOG.md:14]: "pull/51" to "pull/511"

Comment (afontani) [reply CHANGELOG.md]: Oh, man. Thanks for catching this.

**Comment by joseph-robertson** [comment]
@afontani Is this good to go?

**Comment by afontani** [comment]
> @afontani Is this good to go?

Just waiting for post processing on Eagle to check the results.

**Review by afontani — APPROVED** [review]

**Pull request merged**